test/BUILD.bazel

```diff
@@ -14,6 +14,9 @@
         "OPENROAD_EXE": "$(location //:openroad)",
         "TEST_EXT": "tcl",
     },
+    # No point in running these tests with unless they fail,
+    # they would be too slow.
+    tags = ["optonly"],
```

Comment:
Let's see if it works, I think my previous suggestion is something I misremembered.